dsl-executor-service/src/main/java/com/lohika/jclub/dsl/service/DslController.java

```diff
@@ -0,0 +1,54 @@
+package com.lohika.jclub.dsl.service;
+
+import groovy.lang.GroovyShell;
+import groovy.util.DelegatingScript;
+
+import com.lohika.jclub.dsl.MyDsl;
+import com.lohika.jclub.rating.client.RatingServiceClient;
+import com.lohika.jclub.storage.client.StorageServiceClient;
+
+import org.codehaus.groovy.control.CompilerConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@RestController
+@RequestMapping(path = "/dsl")
+public class DslController {
```

Comment:
Why DSL ? We have some specific domain for this script, right? Then lets make this controller represents actual meaning.